typespec-extension/src/code-model-builder.ts

```diff
@@ -2146,14 +2154,39 @@ export class CodeModelBuilder {
 
   private dummyResourceSchema(type: Model, name?: string): ObjectSchema {
```

Comment:
nit, see if you can move this func (and related ones) out of this ts (it is already too big), e.g. type-utils or external-schemas

Reply:
OK, probably not possible.